frontend/pages/SoftwarePage/SoftwareTitles/SoftwareTable/SoftwareTitlesTableConfig.tsx

```diff
@@ -70,13 +70,15 @@ const generateTableHeaders = (
           id.toString()
         )}?${teamQueryParam}`;
 
+        const hasPackage = Boolean(software_package) && !!teamId; // teamId is required for package installation
```

Comment:
why use `Boolean` for one and `!!` for the other?

Reply:
Variety is the spice of life

Reply:
approved